augly/image/transforms.py

```diff
@@ -1770,10 +1774,14 @@ def __init__(
         @param height: the desired height the image should be resized to have. If None,
             the original image height will be used
 
+        @param resample: An optional resampling filter. This can be one of
+            PIL.Image.NEAREST, PIL.Image.BOX, PIL.Image.BILINEAR, PIL.Image.HAMMING,
+            PIL.Image.BICUBIC or PIL.Image.LANCZOS
```

Comment:
```suggestion
        @param resample: A resampling filter. This can be one of PIL.Image.NEAREST,
            PIL.Image.BOX, PIL.Image.BILINEAR, PIL.Image.HAMMING, PIL.Image.BICUBIC, or
            PIL.Image.LANCZOS
```